models/multiphase/d2q9_pf_velocity/Dynamics.R

```diff
@@ -105,3 +105,5 @@ AddGlobal(name="RTISpike",  comment='Spike Tracker')
 # Boundary things
 AddNodeType(name="MovingWall_N", group="BOUNDARY")
 AddNodeType(name="MovingWall_S", group="BOUNDARY")
```

Comment:
SSymmetr NSymmetr etc

Reply:
@mdzik We will soon move to unified naming for symmetries (and automatic symmetries). This will be corrected in all models.